cli/.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
+package-lock.json
```

Comment:
We need to commit package-lock.json let's not add it to .gitignore

Reply:
we don't need to add package-lock.json since its an npm package and npm does not consider it

Reply:
Wouldn't not pinning version dependency make the package behave differently in case the dependency APIs change? 🤔 
Like [nest-cli](https://github.com/nestjs/nest-cli) which does something similar does this.

Reply:
npm does not consider package-lock, but for testing and ci-cd we can make use of it, we can generate `npm-shrinkwrap.json` which does the same role of package-lock.
Discussed the same with @gondar00 and added package-lock, removed it from `gitignore`

Reply:
Oh I see, TIL.